.github/workflows/quality.yml

```diff
@@ -203,3 +203,26 @@ jobs:
           name: detekt-report
           path: |
             */build/reports/detekt/detekt.html
+
+  towncrier:
+    name: Towncrier check
+    runs-on: ubuntu-latest
+    if: always() && github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Install towncrier
+        run: |
+          python3 -m pip install towncrier
+      - name: Run towncrier
+        run: |
+        # Fetch the pull request' base branch so towncrier will be able to
+        # compare the current branch with the base branch.
+        # Source: https://github.com/actions/checkout/#fetch-all-branches.
+        git fetch --no-tags origin +refs/heads/${BASE_BRANCH}:refs/remotes/origin/${BASE_BRANCH}
+        towncrier check --compare-with origin/${BASE_BRANCH}
```

Comment:
```suggestion
          # Fetch the pull request' base branch so towncrier will be able to
          # compare the current branch with the base branch.
          # Source: https://github.com/actions/checkout/#fetch-all-branches.
          git fetch --no-tags origin +refs/heads/${BASE_BRANCH}:refs/remotes/origin/${BASE_BRANCH}
          towncrier check --compare-with origin/${BASE_BRANCH}
```

Reply:
(for indentation to match earlier indentation)

Reply:
If I do that, I am loosing the syntactic coloration, which is strange:

Without indentation:
<img width="340" alt="image" src="https://user-images.githubusercontent.com/3940906/178246103-8f62aac0-75f2-41d8-8c51-ef3cf5df807e.png">

With indentation:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/3940906/178246043-e977e806-4109-407e-a341-1e094fab8209.png">

this is maybe a bug in the editor, but it makes me not confident to do this change...

Reply:
OK, done in 6feca4badf939353fab736bcf173cb684e28e854